### ISSUE TITLE
refactor unify msg

### ISSUE THREAD

**Issue by lispc** [fluidex/dingir-exchange#262]
Step1

1. For deposit / withdraw, we should generate two msgs, one is the old balance msg in 'balance' topic, the other is the 'deposit' msg or 'withdraw' msg in the 'unify' topic.

2. For inner transfer we should generate 3 msgs, 2 are the old balance msg in 'balance' topic(add, sub), the third is the 'transfer' msg  in the 'unify' topic.

Step2 
3. only put 'Cancel' order msg whose traded volume is non-zero into unify topic.   See https://github.com/Fluidex/rollup-state-manager/blob/98d54f4410597e024600570e5297366b4cbb2970/src/msg/msg_processor.rs#L124

in short 
In the persistor bin in this repo, we store balance update msg into db.  

In the rollup side, we only process 'unify' topic.

**Comment by lispc** [comment]
When we finish this refactor, we can launch 'fluidex-backend', then we can make an inner transfer and rollup will not crash. ( like examples/js/transfer.ts )
